frontend/src/components/viewers/MarkdownViewer.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+import Viewer, { ViewerConfig } from './Viewer';
+// tslint:disable-next-line:no-var-requires
+const markdownIt = require('markdown-it');
```

Comment:
Shouldn't this just be a harmony import statement? What made you use `require` here?

Reply:
markdown-it's Typescript types aren't working well with the latest compiler, and it doesn't seem like it's a priority for them to fix it, this is the only import style that stopped the compiler from screaming at me.